core/src/components/setup/RecommendedApps.vue

```diff
@@ -53,12 +53,21 @@
 			</div>
 		</div>
 
-		<InstallButton v-if="showInstallButton"
-			@click.stop.prevent="installApps" />
-
-		<p class="text-center">
-			<a :href="defaultPageUrl">{{ t('core', 'Cancel') }}</a>
-		</p>
+		<div class="dialog-row">
+			<Button type="tertiary-no-background"
```

Comment:
```suggestion
			<Button type="tertiary"
```

No background should only be used inside of inputs

Reply:
This is still not resolved?